android/app/src/bt/java/covidsafepaths/bt/exposurenotifications/nearby/ExposureConfigurations.java

```diff
@@ -29,17 +29,22 @@
  */
 public class ExposureConfigurations {
 
-  private final ExposureNotificationSharedPreferences prefs;
-
-  public ExposureConfigurations(Context context) {
-    prefs = new ExposureNotificationSharedPreferences(context);
-  }
-
-  public ExposureConfiguration get() {
-    return new ExposureConfiguration.ExposureConfigurationBuilder()
-        .setDurationAtAttenuationThresholds(
-            // TODO: Make these settable in debug UI
-            prefs.getAttenuationThreshold1(50), prefs.getAttenuationThreshold2(60))
-        .build();
-  }
+    private final ExposureNotificationSharedPreferences prefs;
+
+    public ExposureConfigurations(Context context) {
+        prefs = new ExposureNotificationSharedPreferences(context);
+    }
+
+    public ExposureConfiguration get() {
+        return new ExposureConfiguration.ExposureConfigurationBuilder()
+                .setMinimumRiskScore(15)
+                .setDurationAtAttenuationThresholds(
+                        // TODO: Make these settable in debug UI
```

Comment:
What do you think about removing this todo and adding a card for this work in trello?

Reply:
That was copied from Google's repository
I'm not sure if we need it since we will get those values from the backend in the future

Reply:
I'll remove the comment